src/main.go

```diff
@@ -211,6 +221,20 @@ func getAsyncFlag() cli.BoolFlag {
 	return cli.BoolFlag{Name: fAsync, Usage: "Asynchronous execution of the jobflow steps"}
 }
 
+func getLockFlag() cli.StringFlag {
+	return cli.StringFlag{
+		Name:  fLock,
+		Usage: "Name of the lock held for the duration of the jobflow steps",
```

Comment:
"of the jobflow steps. Stored in a file or in Consul depending on the --" + fConsul + " flag"